terps/CMakeLists.txt

```diff
@@ -327,7 +327,8 @@ endif()
 if(WITH_MAGNETIC)
     terp(magnetic
         SRCS magnetic/Generic/emu.c magnetic/Glk/glk.c
-        INCLUDE_DIRS magnetic/Generic)
+        INCLUDE_DIRS magnetic/Generic
+        MACROS MAGNETIC_GLKUNIX GARGLK)
```

Comment:
This shouldn't define `GARGLK` since that's already defined in `glk.h`.

Reply:
Do you want me to update the version from 2.3 to 2.3.1 in the code? Or is it better to do it in a different request?